apps/pie-storybook/stories/pie-toast.stories.ts

```diff
@@ -26,6 +26,21 @@ const toastStoryMeta: ToastStoryMeta = {
                 summary: defaultProps.isOpen,
             },
         },
+        variant: {
+            description: 'Set the variant of the notification.',
```

Comment:
```suggestion
            description: 'Set the variant of the toast.',
```

Reply:
I cannot update my branch due conflicts with some automatic updates. I will fix this in another PR